src/jdk.jdi/share/classes/com/sun/jdi/VirtualMachine.java

```diff
@@ -678,7 +678,8 @@ default List<ModuleReference> allModules() {
      * Determines if the target VM supports the filtering of
      * class prepare events by source name.
      *
-     * see {@link ClassPrepareRequest#addSourceNameFilter}.
+     * @see ClassPrepareRequest#addSourceNameFilter
```

Comment:
It is not clear why the link has been removed. There are some other similar places with links but others without links, so it is not clear how to achieve consistency here. Just want to understand your logic.

Reply:
`@see` doesn't require using `@link`. It is implied. The text ends up in the `See Also:` section of the javadoc. The `See Also:` is just a list of links.

You may also see something like the following:

     * @throws VMCannotBeModifiedException if the VirtualMachine is read-only
     * - see {@link VirtualMachine#canBeModified()}.

In this case this is not something that ends up in the `See Also:` section. The text is simply part of the `@throw` description, and there is nothing special about the word `see`. Thus the `@link` is needed.

For this particular case you are commenting on, it seemed more appropriate to put this in the `See Also:` section rather than inlined with the descriptive text since it did not seem to be an extension of the sentence that preceded it.

Reply:
Okay, thanks! Reasonable.